Current time PTO clarification (#3911)
quicwg/base-drafts: draft-ietf-quic-recovery.md

A client could have received and acknowledged a Handshake packet, causing it to
discard state for the Initial packet number space, but not sent any

> ack-eliciting Handshake packets.  In this case, when the Initial packet number
> space is discarded, the PTO is set from the current time.

Reviewer: I would have said instead that:

```suggestion
ack-eliciting Handshake packets.  In this case, the PTO timer starts at the
time that the Initial packet number space is discarded.
```
